### PR TITLE
Imag link/test fixup

### PR DESCRIPTION
**Pull request by matthiasbeyer**
imag-link script tests were broken.

And nobody told me. :cry:

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@mario-kr could you review this?

**Comment by mario-kr** [comment]
> And nobody told me. :cry: 

Well, I didn't notice either.
The change looks fine, but seeing as you had to change 14 lines of code to change a static string, I would suggest to store the name/id of the entries in a variable.

File: imag-link/tests/link-test.sh [hunk 1]
```diff
@@ -31,35 +31,35 @@ mktestentry() {
...
-    mktestentry "test~0.2.0"
-    mktestentry "test2~0.2.0"
+    mktestentry "test"
+    mktestentry "test2"
 
-    imag-link internal add --from "test~0.2.0" --to "test2~0.2.0"
+    imag-link internal add --from "test" --to "test2"
 
-    if [[ "$(entry_linked_to '/test~0.2.0')" == "$(cat_entry 'test2~0.2.0')" ]];
+    if [[ "$(entry_linked_to '/test')" == "$(cat_entry 'test2')" ]];
```
Comment (mario-kr) [inline imag-link/tests/link-test.sh]: I dont know how links look with the new StoreId, but shouldnt there be no slash?
also in line 59

**Comment by matthiasbeyer** [comment]
Fixed your annotation and rebased onto latest master, so we can be sure that everything works.

**Comment by matthiasbeyer** [comment]
@mario-kr review pls!

**Comment by mario-kr** [comment]
code looks fine, the question is if it actually detects faults. Do you have the means to create faulty entries and test it?
That aside, you can merge.

**Comment by matthiasbeyer** [comment]
I just saw that I should rebase and fixup ...

**Pull request merged**